beta-src/src/components/ui/units/WDFleetIcon.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import { useTheme } from "@mui/material/styles";
+import { IconProps } from "../../../interfaces/Icons";
+import UIState from "../../../enums/UIState";
+
+const WDFleetIcon: React.FC<IconProps> = function ({
+  country,
+  iconState = UIState.NONE,
+}): React.ReactElement {
+  const theme = useTheme();
+
+  return (
+    <>
+      {" "}
```

Comment:
Can we remove?

Reply:
Not sure actually, I always assumed this was here for a reason since it's been there since the icon was created.

Reply:
It wasn't needed, I've removed it.